Library/Homebrew/missing_formula.rb

```diff
@@ -9,8 +9,12 @@ module Homebrew
   module MissingFormula
     class << self
       def reason(name, silent: false, show_info: false)
-        cask_reason(name, silent: silent, show_info: show_info) || blacklisted_reason(name) ||
-          tap_migration_reason(name) || deleted_reason(name, silent: silent)
+        if OS.mac?
```

Comment:
Remove `OS.mac?` and make `cask_reason` a no-op instead.

Reply:
This is done.